Makefile

```diff
@@ -206,6 +214,9 @@ endif
 dockertest-sync: dockerbuild-test dockerrun-sync
 .PHONY: dockertest-sync
 
+dockertest-harness: dockerbuild-test dockerrun-harness
```

Comment:
`dockerrun-harness` doesn't exist - what is `dockertest-harness` supposed to do? Can I delete it?